packages/desktop-gui/src/lib/ipc-bus.js

```diff
@@ -96,15 +106,7 @@ const ipcBus = (...args) => {
     }
   }
 
-  // remove nodes that cannot be cloned
-  // https://github.com/cypress-io/cypress/issues/6750
-  args = args.map((arg) => {
-    return _.cloneDeepWith(arg, (val) => {
-      if (_.isFunction(val) || _.isElement(val)) {
-        return null
-      }
-    })
-  })
+  args = nullifyUnserializableValues(args)
```

Comment:
wait, this used to be a callback to `args.map`, right. Does `nullifyUnserializableValues` properly handle arrays? Does it just recurse into them?

Reply:
oh oops, good catch

Reply:
not sure if cloneDeep will work with the array

Reply:
yes, it should still work with the array, since it uses cloneDeep

Reply:
desktop-gui tests will fail if this is wrong